src/website/assets/scripts/algoliaSearch.js

```diff
@@ -0,0 +1,85 @@
+const searchClient = algoliasearch('DI4R9YBA6J', '81112f1dd8329496fd36490bdc8a9aee');
+let timerId;
+
+const printSearchResultItem = (item) => {
+  return `
+    <a class="search_result_a" href="${item.url}">
+      <div class="title">${item.name} <span class="search_result_category">| ${item.category}</span></div>
+      <div class="description">${item.description}</div>
+    </a>
+  `;
+}
+
+const search = instantsearch({
+    indexName: 'components',
+    searchClient,
+    searchFunction(helper) {
+      const aisSearchBoxForm = document.querySelector('.ais-SearchBox-form');
+      const container = document.querySelector('#search_results');
+      const icon = document.createElement('i');
+      const searchInput = document.querySelector('.ais-SearchBox-input');
+      const searchResults = document.querySelector('#search_results');
+
+      icon.classList = 'a-icon icon-search -text--muted';
+      searchInput.type = 'text';
+      
+      searchInput.addEventListener('focus', function() {
+        searchInput.style.width = '370px';
+        searchResults.style.width = '370px';
+      });
+      searchInput.addEventListener('blur', function() {
+        searchInput.style.width = '300px';
+        searchResults.style.width = '300px';
+      });
+
+      if(!aisSearchBoxForm.classList.contains("a-inputWrapper")) {
+        aisSearchBoxForm.classList.add("a-inputWrapper");
+        aisSearchBoxForm.classList.add("-icon--left");
+        aisSearchBoxForm.appendChild(icon);
+      }
+
+      if (helper.state.query.length < 3) {
```

Comment:
@janibolkvadze @jllr @dani-cl-madrid can we reduce to 1? Algolia docsearch defaults to 1 and feels much faster

Reply:
@mattnickles New PR is implemented using Algolia Docsearch. I think it's a much better option for us. Thanks for the suggestions!